bindings/python/src/OpenSpaceToolkitMathematicsPy/Geometry/2D/Objects.cpp

```diff
@@ -8,6 +8,7 @@
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 #include <OpenSpaceToolkitMathematicsPy/Geometry/2D/Objects/MultiPolygon.cpp>
+#include <OpenSpaceToolkitMathematicsPy/Geometry/2D/Objects/Composite.cpp>
```

Comment:
To keep the ordering consistent w/ below:
```suggestion
#include <OpenSpaceToolkitMathematicsPy/Geometry/2D/Objects/Composite.cpp>
#include <OpenSpaceToolkitMathematicsPy/Geometry/2D/Objects/MultiPolygon.cpp>
```